draft-smyslov-ipsecme-ikev2-mceliece-00.xml

```diff
@@ -68,6 +71,7 @@
             necessary.</t>
           <t> TBD if this document should make any recommendations about which DH group to use for the 
             initial key exchange, or whether it should remain silent on the topic.</t>
+        </section>

```

Comment:
Why should `mustshouldmay` be its own top-level section?

Reply:
I think there is no strict rule whether it deserves a top-level section or is a sub-section of Introduction. Do you think the latter is better?